dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/metadata/objectbundle/feedback/ObjectBundleCommitReport.java

```diff
@@ -91,14 +74,14 @@ public void addTypeReport( TypeReport report )
             return;
         }
 
-        if ( !typeReportMap.containsKey( report.getKlass() ) )
-        {
-            typeReportMap.put( report.getKlass(), report );
-            return;
-        }
-
-        TypeReport typeReport = typeReportMap.get( report.getKlass() );
-        typeReport.merge( typeReport );
```

Comment:
Note the bug: `typeReport` is merged with itself instead of merging with `report`.